packages/lesswrong/lib/collections/doppelComments/schema.ts

```diff
@@ -0,0 +1,36 @@
+import { foreignKeyField, schemaDefaultValue } from "@/lib/utils/schemaUtils";
+
+const schema: SchemaType<"DoppelComments"> = {
+  commentId: {
+    ...foreignKeyField({
+      idFieldName: "commentId",
+      collectionName: "Comments",
+      resolverName: "mimickedComment",
+      type: "Comment",
+      nullable: false,
+    }),
+    optional: false,
+    canRead: ["guests"],
+    canCreate: ["admins"],
+    canUpdate: ["admins"],
+  },
+  content: {
+    type: String,
+    optional: false,
+    nullable: false,
+    canRead: ["guests"],
+    canCreate: ["admins"],
+    canUpdate: ["admins"],
+  },
```

Comment:
You explicit want these to be plaintext strings rather than WYSIWYG editable fields?

Reply:
:shrug:. They are programatically generated. I don't know when I would edit them